services/motion/motion_test.go

```diff
@@ -828,29 +828,6 @@ func TestMoveOnGlobeReq(t *testing.T) {
 	name := "somename"
 	dst := geo.NewPoint(1, 2)
 
-	t.Run("String()", func(t *testing.T) {
```

Comment:
This whole test seems fragile and not worth it IMO

Reply:
Agree